lib/remote_storage/swift.rb

```diff
@@ -379,21 +383,17 @@ def redis_metadata_object_key(user, directory, key)
     end
 
     def container_url_for(user)
-      "#{base_url}/#{container_for(user)}"
+      if container_migration(user)
+        user_container_url
```

Comment:
Yes, as discussed we'll have to set the Redis key for all existing users before deploying this.

This way, all new users that register afterwards will automatically use the new shared container. And the old users will only use them after the migration has finished successfully.